### ISSUE TITLE
Accessibility : High contrast mode, menu items not visible

### ISSUE THREAD

**Issue by amitse** [Microsoft/vscode#12581]
- VSCode Version: Code 1.5.3 (5be4091987a98e3870d89d630eb87be6d9bafd27, 2016-09-22T08:50:42.185Z)
- OS Version: Windows_NT ia32 10.0.14393

Steps to Reproduce:

1) Set High contrast black theme ( in windows : Setting -> Ease of Access -> High Contrast)
2) Open Vscode 

Menu items are not visible. They are only visible on hover as shown below.

![vsc](https://cloud.githubusercontent.com/assets/2761516/18817450/1e949cb2-837e-11e6-81d2-8c220eb24adc.png)

**Comment by amitse** [comment]
duplicate of #6629

**Comment by chrmarti** [comment]
Tracked as #6629
